### DISCUSSION TITLE
HyperHDR Detailed Guide

### DISCUSSION THREAD

**Discussion by EverythingSmartHome** [awawa-dev/HyperHDR#77] [Show and tell]
Hello (again!)

Not sure if you are interested or if its useful for your audience but to let you know I just covered the HyperHDR setup in detail here:

https://youtu.be/J26oYlKyq7Q

Hope someone finds it useful!

Lewis

**Comment by maaachew** [comment]
I followed this guide on you tube. I am new to all this so hoped it it be a fairly straight forward install. All good until I came to confirm the RGB and no LEDs lit up. I have checked and double checked my wiring and I belive all is good. Hyper HDR also seems to be installed correctly on the PI. I am using an sk6812 RGBW strip. When I set the controlled in HyperHDR to this setting I get nothing. If I set it to WS281x then I get some random LEDs flashing off and on in various colours, so I can see the LED is responding to an incorrect controller type. Any insight on what possible things to check would be very much appreciated
